tools/schema-generator.py

```diff
@@ -843,6 +843,7 @@ def main():
     model_definition = { "groups": {} }
     for input_file in args.input_files:
         with open(input_file) as file:
+            # print(f"File: {input_file}")
```

Comment:
My guess is this was for debugging purposes - I'd suggest either removing, uncommenting, or making it conditional on some debug flag.

Reply:
this isn't the first time I spent too long trying to figure out where to add this debug line so I decided to make my life easier. But... as payback for all of your time doing these reviews I'll remove it :-)